<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1143</width>
    <height>901</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="cat1_1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$100</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cat2_1">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>150</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$100</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cat1_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$200</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cat4_1">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>150</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$100</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cat5_1">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>150</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$100</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cat3_1">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>150</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$100</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cat1_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>370</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$300</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cat1_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>480</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$400</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cat1_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>590</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$500</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cat2_2">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>260</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$200</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cat3_2">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>260</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$200</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cat4_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>260</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$200</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cat5_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>260</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$200</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cat2_3">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>370</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$300</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cat3_3">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>370</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$300</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cat4_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>370</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$300</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cat5_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>370</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$300</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cat2_4">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>480</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$400</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cat3_4">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>480</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$400</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cat4_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>480</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$400</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cat5_4">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>480</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$400</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cat2_5">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>590</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$500</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cat3_5">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>590</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$500</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cat4_5">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>590</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$500</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cat5_5">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>590</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>$500</string>
    </property>
   </widget>
   <widget class="QLabel" name="cat1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Cat 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="cat2">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>100</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Cat2</string>
    </property>
   </widget>
   <widget class="QLabel" name="cat3">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>100</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Cat3</string>
    </property>
   </widget>
   <widget class="QLabel" name="cat4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>100</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Cat4</string>
    </property>
   </widget>
   <widget class="QLabel" name="cat5">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>100</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Cat5</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1143</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
